src/ert/job_queue/queue.py

```diff
@@ -50,30 +50,30 @@
 
 LONG_RUNNING_FACTOR = 1.25
 
-
-_FM_STEP_FAILURE = "com.equinor.ert.forward_model_step.failure"
-_FM_STEP_PENDING = "com.equinor.ert.forward_model_step.pending"
-_FM_STEP_RUNNING = "com.equinor.ert.forward_model_step.running"
-_FM_STEP_SUCCESS = "com.equinor.ert.forward_model_step.success"
-_FM_STEP_UNKNOWN = "com.equinor.ert.forward_model_step.unknown"
-_FM_STEP_WAITING = "com.equinor.ert.forward_model_step.waiting"
-
+# Duplicated from ert.ensemble_evaluator.identifiers to avoid circular imports
```

Comment:
What about putting those `identifiers.py` into some standalone module?

Reply:
That could probably solve the circular uglyness yes, but I think that should be its own PR :)

Reply:
https://github.com/equinor/ert/issues/6457

Reply:
(the duplication was there prior to this PR)